liveRecorder/KalturaClient/Client.py

```diff
@@ -25,25 +25,25 @@
 #
 # @ignore
 # ===================================================================================================
-from Plugins.Core import *
-from Base import *
+from __future__ import absolute_import
+
+from .Plugins.Core import *
+from .Base import *
 from xml.parsers.expat import ExpatError
 from xml.dom import minidom
-from threading import Timer
-from StringIO import StringIO
+import binascii
 import hashlib
+import mimetypes
 import random
 import base64
-import socket
 import urllib
 import types
-import gzip
 import time
 import os
 
-from poster.streaminghttp import register_openers
-from poster.encode import multipart_encode
-import urllib2
+import requests
+from requests_toolbelt.multipart.encoder import MultipartEncoder
+import six
```

Comment:
not a big fan of adding more dependencies (what is this six library...)
will not need update install scripts + jess....

Rather not go that way